### PR TITLE
Am/chore/cpk params

### PR DESCRIPTION
**Pull request by IceTDrinker**

### CODE REVIEW

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test
@slab-ci cpu_integer_test
@slab-ci cpu_multi_bit_test
@slab-ci cpu_wasm_test

File: .config/nextest.toml [hunk 1]
```diff
@@ -8,10 +8,10 @@ slow-timeout = "5m"
...
 
 [[profile.ci.overrides]]
-filter = 'test(/^.*param_message_1_carry_[567]$/) or test(/^.*param_message_4_carry_4$/)'
+filter = 'test(/^.*param_message_1_carry_[567]_ks_pbs$/) or test(/^.*param_message_4_carry_4_ks_pbs$/)'
 retries = 3
 
 [[profile.ci.overrides]]
-filter = 'test(/^.*param_message_[23]_carry_[23]$/)'
+filter = 'test(/^.*param_message_[23]_carry_[23]_ks_pbs$/)'
 retries = 1
```
Comment (IceTDrinker) [inline .config/nextest.toml:16]: not sure the filters are still needed

**Comment by IceTDrinker** [comment]
muli bit shortint tests did not run properly, to be checked

File: .config/nextest.toml (hunk 1 above)
Comment (IceTDrinker) [reply .config/nextest.toml:16]: looks like this conf file is broken anyways

File: tfhe/src/c_api/shortint/parameters.rs [hunk 2]
```diff
@@ -156,49 +158,49 @@ macro_rules! expose_as_shortint_pbs_parameters(
 );
 
 expose_as_shortint_pbs_parameters!(
```
Comment (IceTDrinker) [inline tfhe/src/c_api/shortint/parameters.rs:160]: missing CPK params

File: tfhe/src/shortint/server_key/mod.rs [hunk 3]
```diff
@@ -239,11 +239,11 @@ impl ServerKey {
     /// # Example
```
Comment (IceTDrinker) [inline tfhe/src/shortint/server_key/mod.rs:239]: no need to remove the big/small examples

File: tfhe/src/shortint/public_key/compressed.rs [hunk 4]
```diff
@@ -22,11 +22,11 @@ impl CompressedPublicKey {
     ///
```
Comment (IceTDrinker) [inline tfhe/src/shortint/public_key/compressed.rs:22]: check docstrings which got shortened

File: tfhe/src/shortint/public_key/standard.rs [hunk 5]
```diff
@@ -22,11 +22,11 @@ impl PublicKey {
     ///
```
Comment (IceTDrinker) [inline tfhe/src/shortint/public_key/standard.rs:22]: check shortened docstrings

Comment (IceTDrinker) [reply tfhe/src/shortint/public_key/standard.rs:22]: looks fine

File: tfhe/src/shortint/server_key/mod.rs (hunk 3 above)
Comment (IceTDrinker) [reply tfhe/src/shortint/server_key/mod.rs:239]: looks fine

File: tfhe/src/c_api/shortint/parameters.rs (hunk 2 above)
Comment (IceTDrinker) [reply tfhe/src/c_api/shortint/parameters.rs:160]: done

**Review by tmontaigu — APPROVED** [review]

File: tfhe/c_api_tests/test_high_level_boolean.c
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/c_api_tests/test_high_level_boolean.c:1]: yeah I'll open an issue, it's my bad I must have hit the fmt shortcut in my IDE

File: tfhe/docs/shortint/parameters.md [hunk 6]
```diff
@@ -8,18 +8,18 @@ All parameter sets provide at least 128-bits of security according to the [Latti
 
 The user is allowed to choose which set of parameters to use when creating the pair of keys.
 
-The difference between the parameter sets is the total amount of space dedicated to the plaintext and how it is split between the message buffer and the carry buffer. The syntax chosen for the name of a parameter is: `PARAM_MESSAGE_{number of message bits}_CARRY_{number of carry bits}`. For example, the set of parameters for a message buffer of 5 bits and a carry buffer of 2 bits is `PARAM_MESSAGE_5_CARRY_2`.
+The difference between the parameter sets is the total amount of space dedicated to the plaintext and how it is split between the message buffer and the carry buffer. The syntax chosen for the name of a parameter is: `PARAM_MESSAGE_{number of message bits}_CARRY_{number of carry bits}`. For example, the set of parameters for a message buffer of 5 bits and a carry buffer of 2 bits is `PARAM_MESSAGE_5_CARRY_2_KS_PBS`.
```
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/docs/shortint/parameters.md:11]: correct, I'll look to add info for perf/ciphertext size for KS_PBS/PBS_KS

File: tfhe/c_api_tests/test_high_level_boolean.c
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/c_api_tests/test_high_level_boolean.c:1]: https://github.com/zama-ai/tfhe-rs-internal/issues/175

File: tfhe/docs/shortint/parameters.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/docs/shortint/parameters.md]: done let me know what you think about the new wording

**Comment by IceTDrinker** [comment]
arf broken rebase

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test
@slab-ci cpu_integer_test
@slab-ci cpu_multi_bit_test
@slab-ci cpu_wasm_test

File: tfhe/docs/shortint/parameters.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tmontaigu) [reply tfhe/docs/shortint/parameters.md]: looks good 👍

**Review by tmontaigu — APPROVED** [review]

**Pull request merged**